velox/exec/tests/AggregationFuzzerTest.cpp

```diff
@@ -80,19 +81,20 @@ int main(int argc, char** argv) {
           {"approx_percentile_partial", ""},
           {"approx_percentile_merge", ""},
           {"arbitrary", ""},
-          {"array_agg", "array_sort({})"},
-          {"array_agg_partial", "array_sort({})"},
-          {"array_agg_merge", "array_sort({})"},
-          {"array_agg_merge_extract", "array_sort({})"},
-          {"set_agg", "array_sort({})"},
-          {"set_union", "array_sort({})"},
-          {"map_agg", "array_sort(map_keys({}))"},
-          {"map_union", "array_sort(map_keys({}))"},
-          {"map_union_sum", "array_sort(map_keys({}))"},
+          {"array_agg", "\"$internal$canonicalize\"({})"},
```

Comment:
Note that this change reduces coverage in the AggregationFuzzer. Before this change, the Fuzzer was able to verify results against DuckDB (since it supports array_sort function), but now it cannot. We are going to face same problem with using Presto as the reference query runner. A solution could be to change the Fuzzer to apply post-aggregation projections using Velox only (once to Velox results, once to results from reference DB), then compare. This is a follow-up though.